python/pyspark/ml/tests.py

```diff
@@ -1027,6 +1027,21 @@ def test_storage_levels(self):
         self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
         self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
 
+    def test_unknown_strategy(self):
```

Comment:
The params on the Java side are only set when fit is called afaik.

But yeah the tests are not strictly necessary. I could add a doc string
test for it. My only concern (as with the storage params) is I didn't want
to highlight what is a bit more of an expertparam

On Wed, 4 May 2016 at 18:05, Seth Hendrickson <notifications@github.com>
wrote:

> In python/pyspark/ml/tests.py
> <https://github.com/apache/spark/pull/12896#discussion_r62066768>:
>
> > @@ -1027,6 +1027,21 @@ def test_storage_levels(self):
> >          self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
> >          self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
> >
> > +    def test_unknown_strategy(self):
>
> I see a similar test was added for the storage level params, but I don't
> exactly understand why we need these. We don't implement these types of
> tests for other classes and params and this is just checking the getters
> and setters. Also, the fit call seems unnecessary.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12896/files/fc437451a598221f0878b7a2e0b87d17572019cc#r62066768>
>